codegen/java-gen/src/test/scala/com/lightbend/akkasls/codegen/java/EventSourcedEntityTestKitGeneratorSuite.scala

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.lightbend.akkasls.codegen
+package testkit
+
+class EventSourcedEntityTestKitGeneratorSuite extends munit.FunSuite {
+
+  test(
+    "it can generate an specific TestKit for the proto files " +
+    "in test/resources/testkit"
+  ) {
+
+    val packageName = "com.example.shoppingcart"
+    val entity = generateShoppingCartEntity
+    val service = generateShoppingCartService(entity)
+    val className = "ShoppingCart"
+
+    val sourceCode = EventSourcedEntityTestKitGenerator.generateSourceCode(
+      service,
+      entity,
+      packageName,
+      className
+    )
+
+    val expected =
+      """
+            |package com.example.shoppingcart.domain;
+            |
+            |import com.example.shoppingcart.domain.ShoppingCart;
+            |import com.example.shoppingcart.domain.ShoppingCartDomain;
+            |import com.example.shoppingcart.ShoppingCartApi;
+            |import com.akkaserverless.javasdk.eventsourcedentity.EventSourcedEntityBase;
+            |import com.akkaserverless.javasdk.impl.effect.MessageReplyImpl;
+            |import com.akkaserverless.javasdk.impl.effect.SecondaryEffectImpl;
+            |import com.akkaserverless.javasdk.impl.eventsourcedentity.EventSourcedEntityEffectImpl;
+            |import com.akkaserverless.javasdk.testkit.AkkaserverlessTestKit;
+            |import com.akkaserverless.javasdk.testkit.Result;
+            |import com.google.protobuf.Empty;
+            |import java.util.ArrayList;
+            |import java.util.List;
+            |import java.util.NoSuchElementException;
+            |import scala.jdk.javaapi.CollectionConverters;
+            |
+            |public class ShoppingCartTestKit {
+            |
+            |    private ShoppingCartDomain.Cart state;
+            |    private ShoppingCart entity;
+            |    private List<Object> events = new ArrayList<Object>();
+            |    private AkkaserverlessTestKit helper = new AkkaserverlessTestKit<ShoppingCartDomain.Cart>();
```

Comment:
We haven't been very consistency with it, but I think that most of the generated code and the SDK code is using indent 2 for java.

Reply:
good catch. I'll indent it like so.